package.json

```diff
@@ -115,8 +115,8 @@
         "eslint": "8.54.0",
         "http-server": "14.1.1",
         "husky": "8.0.3",
-        "jest": "28.1.3",
-        "jest-environment-jsdom": "28.1.3",
+        "jest": "29.7.0",
+        "jest-environment-jsdom": "29.7.0",
```

Comment:
It requires Typescript `4.5.0`.
Read more: https://jestjs.io/docs/upgrading-to-jest29#jsdom-upgrade

___

This upgrade will be done later in **Maskito 2.0**